pkg/convert/convert.go

```diff
@@ -25,21 +27,25 @@ func CsvToResources(resourcesFile string) ([]map[string]interface{}, error) {
 	// Parse resources from CSV
 	for _, record := range records {
 		resource := make(map[string]interface{})
-		resource["id"] = record[0]
-		resource["generic_name"] = record[1]
-		resource["name"] = record[2]
-		resource["cloud_provider_id"] = record[3]
-		resource["cpu_cores"] = record[4]
-		resource["memory"] = record[5]
-		resource["memory_pretty"] = record[6]
-		resource["category"] = record[7]
-		resource["category_pretty"] = record[8]
-		resource["size_pretty"] = record[9]
-		resource["resource_type"] = record[10]
-		resource["active"] = record[11]
+		resource[Id] = record[0]
+		resource[GenericName] = record[1]
+		resource[Name] = record[2]
+		resource[CloudProviderId] = record[3]
+		resource[CpuCores] = record[4]
+		resource[Memory] = record[5]
+		resource[MemoryPretty] = record[6]
+		resource[Category] = record[7]
+		resource[CategoryPretty] = record[8]
+		resource[SizePretty] = record[9]
+		resource[ResourceType] = record[10]
+		resource[Active] = record[11]
 		finalResources = append(finalResources, resource)
 	}
```

Comment:
I would prefer to write this differently, but feel free to ignore this comment if you prefer the current version

```go
	keys := []string{Id, GenericName, Name, CloudProviderId, CpuCores, Memory, MemoryPretty, Category, CategoryPretty, SizePretty, ResourceType, Active}

	// Parse resources from CSV
	for _, record := range records {
		resource := make(map[string]interface{})
		for i, key := range keys {
			resource[key] = record[i]
		}

		finalResources = append(finalResources, resource)
	}
```